introduction.md

```diff
@@ -0,0 +1,3 @@
+Here I am increasing my working knowledge of git & GitHub by taking courses on CodeSchool.  I really enjoyed the Git Real & Git Real 2 courses, so I decided to take Mastering GitHub as well.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.